test/distributed/org/apache/cassandra/fuzz/topology/HarryTopologyMixupTest.java

```diff
@@ -101,28 +108,73 @@ protected void destroyState(State<Spec> state, @Nullable Throwable cause)
+                                                                                                    {
+                                                                                                        int selected = (int) (cnt.getAndIncrement() % cluster.size() + 1);
+                                                                                                        if (!cluster.get(selected).isShutdown())
+                                                                                                            return selected;
+                                                                                                    }
+                                                                                                    throw new IllegalStateException("Unable to find an alive instance");
+                                                                                                }
+                                                                                            })
+                                                                                            .retryPolicy(t -> {
+                                                                                                t = Throwables.getRootCause(t);
+                                                                                                if (!TIMEOUT_CHECKER.matches(t))
+                                                                                                    return false;
+
+                                                                                                TxnId id;
+                                                                                                try
+                                                                                                {
+                                                                                                    id = TxnId.parse(t.getMessage());
+                                                                                                }
+                                                                                                catch (Throwable t2)
+                                                                                                {
+                                                                                                    return true;
+                                                                                                }
+                                                                                                try
+                                                                                                {
+                                                                                                    var nodes = cluster.stream().filter(i -> !i.isShutdown()).mapToInt(i -> i.config().num()).toArray();
```

Comment:
remove `var`, see ML for why

Reply:
Did we land on allowing it only in test code?

Reply:
I never use var, so this must have been a code copied over.

Reply:
https://github.com/apache/cassandra/blob/cep-15-accord/test/distributed/org/apache/cassandra/fuzz/topology/HarryTopologyMixupTest.java#L213

Reply:
Fixed it to be `int`